src/utils/appendToHead.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+/**
+ * Append the element below the last found styled component style tag.
+ * If no tags are found default to appending to the bottom of the head tag.
+ * This prevents any side effects from pre rendered pages or when styles are added after first render.
+ */
+export default function appendToHead(el: HTMLStyleElement) {
+  const scStyleTags = Array.from(
+    document.querySelectorAll('style[data-styled-components]')
+  )
+  if (scStyleTags.length > 0) {
+    const lastScStyleTag = scStyleTags[scStyleTags.length - 1]
+    // $FlowFixMe
+    lastScStyleTag.parentNode.insertBefore(el, lastScStyleTag.nextSibling)
```

Comment:
will this work if the style tag is the last child in its parent context?

Reply:
Probably not; not sure what the alternative is? This maybe: https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement